packages/components/tests/dummy/app/templates/components/tabs.hbs

```diff
@@ -39,6 +39,8 @@
   <p class="dummy-paragraph" id="component-api-tabs">Here is the API for the main (“container”) component:</p>
 
   <dl class="dummy-component-props" aria-labelledby="component-api-tabs">
+    <dt>onTabClick <code>function</code></dt>
+    <dd>Pass an optional function which is called when a Tab is clicked.</dd>
```

Comment:
Let me know if I should make changes to what this arg is called or its functionality

Reply:
I've added a comment above.

Reply:
Thanks!